PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
+  var tableView: UITableView!
```

Comment:
We instantiate this _tableView_ in `viewDidLoad`, so we may do this with lazy init: no implicitly unwrapped optionals, no extra code in `viewDidLoad`
It'll be instantiated when you do anything with the variable for the first time, for example with `view.addSubview`
```suggestion
  private lazy var tableView: UITableView = {
    let tableView = UITableView(frame: self.view.bounds)
    tableView.dataSource = self
    tableView.delegate = self
    return tableView
 }()
```

Reply:
Done